components/Tabs/Tabs.js

```diff
@@ -2,50 +2,45 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
-    
+    this.element= element;    
     // Get the custom data attribute on the Link
-    // this.data;
+     this.data = this.element.dataset;
     
```

Comment:
To clarify, this instruction was actually asking for the value of the `data-tab` attribute (rather than the entire `dataset`). You can access it here directly as `this.element.dataset.tab`.